content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination-sds/index.md

```diff
@@ -0,0 +1,746 @@
+            subset: nginx
+            port:
+              number: 443
+          weight: 100
+      - match:
+        - gateways:
+          - istio-egressgateway
+          port: 443
+        route:
+        - destination:
+            host: my-nginx.mesh-external.svc.cluster.local
+            port:
+              number: 443
+          weight: 100
+    EOF
+    {{< /text >}}
+
+1.  Add a `DestinationRule` to perform simple TLS origination
+
+    {{< text bash >}}
+    $ kubectl apply -n istio-system -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: DestinationRule
+    metadata:
+      name: originate-tls-for-nginx
+    spec:
+      host: my-nginx.mesh-external.svc.cluster.local
+      trafficPolicy:
+        loadBalancer:
+          simple: ROUND_ROBIN
+        portLevelSettings:
+        - port:
+            number: 443
+          tls:
+            mode: SIMPLE
+            credentialName: client-credential # this must match the secret created earlier without the "-cacert" suffix
+            sni: my-nginx.mesh-external.svc.cluster.local
+    EOF
+    {{< /text >}}
+
+1.  Send an HTTP request to `http://my-nginx.mesh-external.svc.cluster.local`:
+
+    {{< text bash >}}
+    $ kubectl exec -it "$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})" -c sleep -- curl -s http://my-nginx.mesh-external.svc.cluster.local
+    <!DOCTYPE html>
+    <html>
+    <head>
+    <title>Welcome to nginx!</title>
+    ...
+    {{< /text >}}
+
+1.  Check the log of the `istio-egressgateway` pod for a line corresponding to our request.
+    If Istio is deployed in the `istio-system` namespace, the command to print the log is:
+
+    {{< text bash >}}
+    $ kubectl logs -l istio=egressgateway -n istio-system | grep 'my-nginx.mesh-external.svc.cluster.local' | grep HTTP
+    {{< /text >}}
+
+    You should see a line similar to the following:
+
+    {{< text plain>}}
+    [2018-08-19T18:20:40.096Z] "GET / HTTP/1.1" 200 - 0 612 7 5 "172.30.146.114" "curl/7.35.0" "b942b587-fac2-9756-8ec6-303561356204" "my-nginx.mesh-external.svc.cluster.local" "172.21.72.197:443"
+    {{< /text >}}
+
+Istio supports reading a few different Secret formats, to support integration with various tools such as [cert-manager](/docs/ops/integrations/certmanager/):
+
+* For a CA only certificate, a generic Secret named `<secret>-cacert`, with a `cacert` key. For example, `httpbin-credential-cacert` has `cacert`.
+* A TLS Secret with keys `tls.key` and `tls.crt`. For CA certificate. For CA certificate, a separate generic Secret named `<secret>-cacert`, with a `cacert` key. For example, `httpbin-credential` has `key` and `cert`, and `httpbin-credential-cacert` has `cacert`.
+* A generic Secret with keys `key` and `cert`. For CA certificate, a separate generic Secret named `<secret>-cacert`, with a `cacert` key. For example, `httpbin-credential` has `key` and `cert`, and `httpbin-credential-cacert` has `cacert`.
```

Comment:
What exactly is this trying to say, and why is it here, instead of in the section where the secret is created?

Reply:
Right this should be moved to secret creation section instead, also the wording is a bit confusing. I'll change up the sentences a bit